recipe-server/client/control_new/components/recipes/Listing.js

```diff
@@ -0,0 +1,185 @@
+    ordering: PropTypes.string,
+    pageNumber: PropTypes.number,
+    push: PropTypes.func.isRequired,
+    recipes: PropTypes.object.isRequired,
+    searchText: PropTypes.string,
+    status: PropTypes.string,
+  };
+
+  static columnRenderers = {
+    name({ ordering }) {
+      return (
+        <Table.Column
+          title="Name"
+          dataIndex="name"
+          key="name"
+          render={Listing.renderLinkedText}
+          sortOrder={DataList.getSortOrder('name', ordering)}
+          sorter
+        />
+      );
+    },
+
+    action() {
+      return (
+        <Table.Column
+          title="Action"
+          dataIndex="action"
+          key="action"
+          render={Listing.renderLinkedText}
+        />
+      );
+    },
+
+    enabled({ status }) {
+      return (
+        <Table.Column
+          title="Enabled"
+          key="status"
+          render={(text, record) => <BooleanIcon value={record.enabled} />}
+          filters={[
+            { text: 'Enabled', value: 'enabled' },
+            { text: 'Disabled', value: 'disabled' },
+          ]}
+          filteredValue={status}
+          filterMultiple={false}
+        />
+      );
+    },
+
+    lastUpdated({ ordering }) {
+      return (
+        <Table.Column
+          title="Last Updated"
+          key="last_updated"
+          dataIndex="last_updated"
+          render={(text, record) => {
+            const lastUpdated = moment(record.last_updated);
+            return (
+              <Link href={`/recipes/${record.id}`} title={lastUpdated.format('LLLL')}>
+                {lastUpdated.fromNow()}
+              </Link>
+            );
+          }}
+          sortOrder={DataList.getSortOrder('last_updated', ordering)}
+          sorter
+        />
+      );
+    },
+  }
+
+  static renderLinkedText(text, record) {
+    return <Link href={`/recipes/${record.id}`}>{text}</Link>;
+  }
+
+  getFilters() {
+    const { ordering, searchText, status } = this.props;
+
+    const filters = {
+      text: searchText,
+      ordering,
+      status,
+    };
+
+    Object.keys(filters).forEach(key => {
+      if (filters[key] === undefined) {
+        delete filters[key];
+      }
+    });
+
+    return filters;
+  }
+
+  handleChangePage(page) {
+    const { getCurrentURL, push } = this.props;
+    push(getCurrentURL({ page }));
+  }
+
+  handleRowClick(record) {
+    const { push } = this.props;
+    push(`recipes/${record.id}`);
+  }
+
+  render() {
+    const { columns, count, ordering, pageNumber, recipes, status } = this.props;
+
+    return (
+      <div>
+        <QueryRecipes
+          pageNumber={pageNumber}
+          filters={this.getFilters()}
+        />
+        <LoadingOverlay>
+          <ListingActionBar />
+
+          <DataList
+            columns={columns}
+            columnRenderers={Listing.columnRenderers}
+            dataSource={recipes.toJS()}
+            ordering={ordering}
+            onRowClick={this.handleRowClick}
```

Comment:
Since we use `onRowClick`, we should set `cursor: pointer;` on rows to denote to the user they can be clicked.